src/Neo.VM/ExecutionEngineLimits.cs

```diff
@@ -36,12 +36,12 @@ public sealed record ExecutionEngineLimits
         /// <summary>
         /// The maximum size of an item in the VM.
         /// </summary>
-        public uint MaxItemSize { get; init; } = 1024 * 1024;
+        public uint MaxItemSize { get; init; } = ushort.MaxValue;
```

Comment:
```suggestion
        public uint MaxItemSize { get; init; } = ushort.MaxValue * 2;
```

Reply:
hhahhaahhh~

Reply:
@shargon yes. it's better than `65535` for me.